astropy/cosmology/io/utils.py

```diff
@@ -62,11 +66,26 @@ def convert_parameter_to_model_parameter(parameter, value, meta=None):
     `astropy.modeling.Parameter`
     """
     # Get from meta information relavant to Model
-    extra = {k: v for k, v in (meta or {}).items()
-             if k in ('getter', 'setter', 'fixed', 'tied', 'min', 'max',
-                      'bounds', 'prior', 'posterior')}
+    extra = {
+        k: v
+        for k, v in (meta or {}).items()
+        if k
+        in (
+            'getter',
+            'setter',
+            'fixed',
+            'tied',
+            'min',
+            'max',
+            'bounds',
+            'prior',
+            'posterior',
+        )
+    }
```

Comment:
```suggestion
    attrs = ('getter', 'setter', 'fixed', 'tied', 'min', 'max', 'bounds', 'prior', 'posterior')
    extra = {k: v for k, v in (meta or {}).items() if k in attrs}
```

Reply:
black reformats your suggestion to:
```python
    attrs = (
        'getter',
        'setter',
        'fixed',
        'tied',
        'min',
        'max',
        'bounds',
        'prior',
        'posterior',
    )
    extra = {k: v for k, v in (meta or {}).items() if k in attrs}
```

Do you still want to keep it?

Reply:
Yeah. It separates the logic better.